creator-node/src/routes/tracks.js

```diff
@@ -480,7 +484,15 @@ module.exports = function (app) {
       return errorResponseBadRequest(`No file found for blockchainId ${blockchainId}`)
     }
 
+    if (libs.identityService) {
+      req.logger.info(`Logging listen for track ${blockchainId} by ${delegateOwnerWallet}`)
+      // Fire and forget listen recording
+      libs.identityService.logTrackListen(blockchainId, delegateOwnerWallet)
```

Comment:
since we don't have a userId or a unauthenticatedUuid we're going to just aggregate these by delegate owner wallet?

Reply:
Yeah -- we need a separate task to organize that a bit. Right now, it doesn't get added to the UserTrackListens table at all (we don't do it for unauthenticatedUuids either) b/c it's an integer type field. But we can be slow about rolling out something there + breaking into the ipwhitelist restriction for these.

Reply:
prob longer term better to add this to queue with retries to ensure this happens? maybe just put a TODO?